sdk/metric/sdk.go

```diff
@@ -291,6 +298,15 @@ func (s *syncInstrument) Bind(kvs []label.KeyValue) api.BoundSyncImpl {
 }
 
 func (s *syncInstrument) RecordOne(ctx context.Context, number api.Number, kvs []label.KeyValue) {
+	if s.meter.metricsLabelsEnricher != nil {
+		var err error
+		kvs, err = s.meter.metricsLabelsEnricher(ctx, kvs)
```

Comment:
This looks good for the unbound instrument code path. 👍 

For the bound code path, in:
```
func (r *record) RecordOne(ctx context.Context, num number.Number) 
```
If the enricher is present, take a new code path, where you'll call
```
s.meter.metricsLabelsEnricher(ctx, record.labels.ToSlice())
```
to get the enriched labels and then divert to the unbound instrument code path, since the `*record` that was bound is not applicable after enrichment of its labels.